src/data_collector/polygon_fundamentals/monitor.py

```
@@ -153,5 +153,6 @@ def __del__(self):
         """Destructor to ensure cleanup"""
         try:
             self.close()
-        except Exception:
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)